src/views/view.c

```diff
@@ -1222,6 +1226,85 @@ int dt_view_image_expose(dt_view_image_over_t *image_over, uint32_t imgid, cairo
   return missing;
 }
 
+void
```

Comment:
I do not really like this.
If this is _really_ needed, *i* would prefer if you would add/pass another param to dt_view_image_expose() that would force-disable display of meta-data.